zebra/zebra_srv6_vty.c

```diff
@@ -233,6 +233,32 @@ DEFUN_NOSH (srv6_locator,
 	return CMD_SUCCESS;
 }
 
+DEFUN (no_srv6_locator,
+       no_srv6_locator_cmd,
+       "no locator WORD",
+       NO_STR
+       "Segment Routing SRv6 locator\n"
+       "Specify locator-name\n")
+{
+	struct srv6_locator *locator = zebra_srv6_locator_lookup(argv[2]->arg);
+	if (!locator) {
+		vty_out(vty, "%% Can't find SRv6 Locator\n");
+		return CMD_WARNING_CONFIG_FAILED;
+	}
+
+	struct listnode *n;
+	struct srv6_locator_chunk *c;
```

Comment:
Can you move variable initialization at the top of the function?

Reply:
Ah, yes I just move them at the top.

Reply:
Updated.